Demo/Demo/Models/CreateOrderParams.swift

```diff
@@ -1,13 +1,16 @@
 struct CreateOrderParams: Codable {
+
     let intent: String
     let purchaseUnits: [PurchaseUnit]
 }
 
 struct PurchaseUnit: Codable {
+
     let amount: Amount
 }
```

Comment:
Hm ... I think the space helps when the file is larger. For Codable structs where are pretty straightforward, it's less helpful.

I don't feel too strongly to be honest. Maybe we could leave the rule as optional, and then just use it when the file is large?

Reply:
I can also update this rule to not apply to codable/decodable structs if that's the main sticking point. 🤷‍♂️

Reply:
Actually looking at my above comment again, it looks like most of our other Codable/Decodable structs (such as Address and Card) already use this spacing (really the demo app is the biggest place this may "look weird"). I think I still am not fully understanding why we wouldn't want a linter rule for this but want a linter rule for other similar styling agreements that we have come to? Would love to make a decision one way or the other on this though. @jonathajones @minhthenguyen @scannillo

Reply:
I am not married to either having/not having this, fwiw. I had some free time during open dev that I wanted to spend understanding more regex and know this styling inconsistency was mentioned in another PR as an item to address, so wanted to try to tackle it since we would need a custom regex rule for it. 😄 More-so just trying to better understand when a rule is needed and when it's not and we are fine with inconsistencies even though we may lean towards a particular style preference.

Reply:
>I think I still am not fully understanding why we wouldn't want a linter rule for this but want a linter rule for other similar styling agreements that we have come to?

I think for me the difference is that I don't think we would necessarily want to follow this for a data container that is >10 lines long (at least in my opinion). For long files, I have no issue with it - but it visually breaks up really small types in a way that looks weird and makes me think we should maybe not have a rule for it (or at least not one that is enforced for small files. Is swiftlint smart enough to enforce only for longer files?)

Reply:
That makes sense, sorry I wasn't following so just wanted to clarify. I am not sure if that's possible! Let me dig into it and see what I can find.

Reply:
Synced with Jon yesterday (and Sammy the day before) and it seems like customizing the regex may be more effort than it's worth and potentially difficult to maintain for other edge cases. Jon and I discussed leaving this rule as is for now and disabling it in places where things look weird. This has been updated, but we both agreed that we should probably revisit this in a few months to see how we are feeling about this rule.